dynamixel_controllers/nodes/controller_manager.py

```diff
@@ -192,78 +222,106 @@ def start_controller(self, req):
         module_name = req.module_name
         class_name = req.class_name
         controller_name = req.controller_name
-        
+
         self.start_controller_lock.acquire()
-        
+
         if controller_name in self.controllers:
             self.start_controller_lock.release()
-            return StartControllerResponse(False, 'Controller [%s] already started. If you want to restart it, call restart.' % controller_name)
-            
+            return StartControllerResponse(
+                False,
+                "Controller [%s] already started. If you want to restart it, call restart."
+                % controller_name,
+            )
+
         try:
             if module_name not in sys.modules:
                 # import if module not previously imported
-                package_module = __import__(package_path, globals(), locals(), [module_name], -1)
+                package_module = __import__(package_path, fromlist=[module_name])
```

Comment:
This whole file changed significantly, but this is the crux of the fix -- python3 doesn't allow for values less than 1 for the relative package hierarchy. Note that we can't use `importlib.import_module`, because catkin modifies the sys.path to perform python package resolution.